draft-ietf-httpbis-client-hints.md

```diff
@@ -130,7 +130,7 @@ Further, depending on the used hint, the server can emit additional response hea
 
 ### Advertising Support for Client Hints {#accept-ch}
 
-Servers can advertise support for Client Hints using the Accept-CH header field or an equivalent HTML meta element with http-equiv attribute ({{W3C.REC-html5-20141028}}).
+Servers should advertise support for Client Hints using the Accept-CH header field or an equivalent HTML meta element with http-equiv attribute ({{W3C.REC-html5-20141028}}).
```

Comment:
"should" is a special word in the IETF -- do you mean SHOULD (i.e., an RFC2119 requirement)? If not, "can" is better. 

SHOULD is a bit problematic here, because typically when you use it, you should define the circumstances when the requirement can be violated.